Makefile

```diff
@@ -21,6 +21,12 @@ bin/network-setup:
 .PHONY: network-setup
 network-setup: bin/network-setup
 
+bin/wsl-proxy:
+	GOOS=linux go build $(LDFLAGS) -o $@ ./cmd/proxy
```

Comment:
Hmm, it's kind of odd that the directory named `proxy` produces a binary named `wsl-proxy` (instead of `proxy`).  But that's an issue we also have with all the other commands…